minidump-stackwalk/src/main.rs

```diff
@@ -373,6 +384,23 @@ async fn main() {
 
     match Minidump::read_path(minidump_path) {
         Ok(dump) => {
+            let mut stdout;
+            let mut output_f;
+            let cyborg_output_f = cyborg.map(|path| File::create(path).unwrap());
+
+            let mut output: &mut dyn Write = if let Some(output_path) = output_file {
+                output_f = File::create(output_path).unwrap();
+                &mut output_f
+            } else {
+                stdout = std::io::stdout();
+                &mut stdout
+            };
+
+            // minidump_dump mode
+            if raw_dump {
+                return print_minidump_dump(&dump, &mut output).unwrap();
```

Comment:
Do you really want this to respect the `--cyborg` option? Feels like it ought to be separate. I'd also suggest pulling in the [`pager`](https://crates.io/crates/pager) crate ([using it is a 1-liner](https://github.com/luser/ppjson/blob/977264905900c96abd672263f8eaed41d592dbaa/src/main.rs#L48)) to get user-friendly output here instead of dumping multiple screens' worth of output to a terminal.